lib/gui/components/update-notifier/services/update-notifier.js

```diff
@@ -34,7 +48,7 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_TIME, M
    *   console.log(`The latest version is: ${latestVersion}`);
    * });
    */
-  this.getLatestVersion = () => {
+  this.getLatestVersion = memoizeVersion(() => {
```

Comment:
I propose to add a JSDoc description to this function explaining that we memoize this function, the reasons we do so.